client/src/components/mixins/common-mixin.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component
+export default class CommonMixin extends Vue {
+  toTitleCase (str: any): string {
```

Comment:
Maybe just a small comment would be cool :)

Reply:
Done.